src/views/Delegate.vue

```diff
@@ -101,7 +111,54 @@ function clearDelegate(id, delegate) {
   modalOpen.value = true;
 }
 
-const { profiles, addressArray } = useProfiles();
+async function getDelegatesWithScore() {
+  delegatesLoading.value = true;
+  try {
+    const { delegations } = await getDelegatesBySpace(
+      space.value.network,
+      space.value.id
+    );
+
+    const uniqueDelegators = Array.from(
+      new Set(delegations.map(d => d.delegate))
+    ).map(delegate => {
+      return delegations.find(a => a.delegate === delegate);
+    });
+
+    const provider = getProvider(space.value.network);
+    const delegatesAddresses = uniqueDelegators.map(d => d.delegate);
+    const delegationStrategy = extentedSpaces.value
+      .find(s => s.id === form.value.id)
```

Comment:
Could this be undefined?

Reply:
I don't see how it could https://github.com/snapshot-labs/snapshot/blob/010e176743cdd5067c15e5036b8f046c01dba6d6/src/views/Delegate.vue#L180